### PR TITLE
Make default urls point to localhost

### PR DESCRIPTION
**Pull request by mbouron**
Hello,

This PR makes the default urls point to localhost instead of jamesbarnsley.co.nz so newly configured instances do not make http requests to a domain they have no control over.

### CODE REVIEW

**Comment by jaedb** [comment]
What is the background for this change?

I host the proxy for authentication of these external OAuth providers (Spotify, LastFM, etc), and the way this works requires a predefined and static return URL. Further to this, an 'app' has been created in each provider to facilitate this authentication.

To help make Iris useful for everyone, these values can be changed to whatever you wish. Advanced users can got through those extra steps if they know what they're doing.

Basically to accept this change would add significant complexity to new users getting started. And if I'm honest, I don't see the point.

**Comment by mbouron** [comment]
> What is the background for this change?

IMHO, default settings should not rely on "third-party" services (even if it is your own). The users have no control on what is happening on it. Having the default installation set to localhost ensure that no query can go somewhere else other than the machine the service is installed on (and makes sure that no information is leaked).

NOTE: I am not implying your service is insecure nor you are doing malicious stuff.

> 
> I host the proxy for authentication of these external OAuth providers (Spotify, LastFM, etc), and the way this works requires a predefined and static return URL. Further to this, an 'app' has been created in each provider to facilitate this authentication.
> 
> To help make Iris useful for everyone, these values can be changed to whatever you wish. Advanced users can got through those extra steps if they know what they're doing.
> 
> Basically to accept this change would add significant complexity to new users getting started. And if I'm honest, I don't see the point.

I understand it greatly helps users and simplify the setup of mopidy-iris but I would set the defaults to localhost and add a section in the documentation to recommend your server if the user wish to use spotify, lastfm, etc.

**Comment by jaedb** [comment]
I hear what you're saying, and agree that it is important that new users understand how things work, especially when external communications are concerned.

I believe the best course of action is to make it abundantly clear what these configuration values mean and what they do. As well as documenting how to replace them with your own custom OAuth connection.

**Comment by jaedb** [comment]
Documentation explains these configuration values, enabling users who wish to host their own authentication scripts to do so. There are also template files for users to get started with; so I believe this addresses the main concerns of this PR.
